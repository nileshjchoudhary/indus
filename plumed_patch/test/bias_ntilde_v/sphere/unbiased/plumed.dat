# PLUMED input file

indus: INDUS INPUTFILE=indus.input TOL=1000.0

#restraint: RESTRAINT ARG=indus.ntilde AT=0.0 KAPPA=1.0

PRINT ...
	LABEL=print
	ARG=indus.n,indus.ntilde
	STRIDE=1
	FILE=plumed.out
... PRINT
