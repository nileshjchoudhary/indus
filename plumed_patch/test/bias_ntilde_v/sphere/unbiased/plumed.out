#! FIELDS time indus.n indus.ntilde
 0.000000 132.000000 132.572883
 1.000000 137.000000 137.504776
 2.000000 137.000000 136.184506
 3.000000 137.000000 138.490494
 4.000000 137.000000 135.224234
 5.000000 135.000000 133.666130
 6.000000 132.000000 133.178801
 7.000000 138.000000 137.655269
 8.000000 136.000000 136.116336
 9.000000 134.000000 133.758068
 10.000000 135.000000 135.357360
